anago

```diff
@@ -920,6 +920,13 @@ if [[ -n "$PARENT_BRANCH" ]]; then
   logecho "$ATTENTION: $RELEASE_BRANCH is a NEW branch off $PARENT_BRANCH!"
 fi
 
+if ! ((FLAGS_nomock)); then
+  logecho
+  logecho "$ATTENTION: This is a mock (--mock) run." \
+          "Publishing will be based on the above values for" \
```

Comment:
Do we publish anything for mock releases? I would assume that mock doesn't have any effects...